Samples/multiple-issuer/Controllers/ApiController.cs

```diff
@@ -20,9 +20,9 @@ public IActionResult Public()
         [HttpGet]
         [Route("private")]
         [Authorize]
-        public IActionResult Private()
+        public ActionResult Private()
         {
-            return Json(new
+            return Ok(new
```

Comment:
I think we should probably leave these as JSON.

Reply:
Whoops, I was testing something and forgot to change that back, will do.